### DISCUSSION TITLE
Making RWs again and again

### DISCUSSION THREAD

**Discussion by ciompek** [isid0re/SoloLeveling#179] [Ideas]
Hi i know that the goal of making the runewords is to make a 'perfect' one but from my perspective, especially in early ladder it is pure waste of runes. If it is not a big deal - i'd like to have the possibility not to make the RWS all over again. For example - in early ladder I dont care if the bot uses 28% fcr spirit because it is temporary solution for my hammerdin. I would prefer to have the bot to make spirit and keep runes so that I can roll spirits myself. Especially - that the shitty spirits make value on d2jsp in early ladder phase.

**Comment by isid0re** [comment]
It's not that it's looking for a perfect spirit. Its just constantly looking to upgrade its gear regardless. If the tier gets above a threshold in the config it wont make that runeword anymore.

As for saving runes to make it yourself. You need to add a pickit for that. You will run into problems with space in the inventory though.

**Comment by ciompek** [comment]
Meaning that if i set the desired parameters of rws it will not make them?

pt., 2 kwi 2021, 06:11 użytkownik isid0re ***@***.***>
napisał:
